elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -1220,6 +1221,7 @@ public Resource toResource(final Supplier<Map<String, Relationship>> relationshi
                 : dictionary.getId(obj));
         resource.setRelationships(relationshipSupplier.get());
         resource.setAttributes(attributeSupplier.get());
+        resource.setLinks(getResourceLevelLink());
```

Comment:
links as a query param seems reasonable to me.  I don't see anything in jsonapi that would preclude it.